src/Umbraco.Tests.UnitTests/Umbraco.Web.BackOffice/Filters/FilterAllowedOutgoingContentAttributeTests.cs

```diff
@@ -78,10 +77,7 @@ public void GetValueFromResponse_Returns_Null_Not_Found_Property()
         [Test]
         public void Filter_On_Start_Node()
         {
-            var userMock = MockedUser.GetUserMock();
-            userMock.Setup(u => u.Id).Returns(9);
-            userMock.Setup(u => u.StartContentIds).Returns(new[] { 5 });
-            var user = userMock.Object;
+            var user = CreateUser(id: 9, startContentId: 5);
```

Comment:
```suggestion
            var user = new UserBuilder()
                .WithId(9)
                .WithStartContentIds(new[] { 5 })
                .Build();
```

IMO, this is a good example where it is an anti-pattern not using the builder directly. Because we build a model that is more complex than needed for the specific tests. 
That is why I like the builder pattern because you can have a minimal model, with just the changes required for the test.

None of the tests in this class seems to need the user group to be set?

Reply:
They don't no - have removed.